app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+from shutil import copy
+
+
+def copy_file(command: str) -> None:
+    comm_data = command.split()
```

Comment:
you can unpack these variables in the easiest way
`command, old_file, new_file = command.split()` 
so there is won't be need to use indexes later